test/test.googleauth.ts

```diff
@@ -1176,7 +1176,8 @@ describe('.getApplicationDefault', () => {
        // tslint:disable-next-line no-any
        assert.equal(null, (result3 as any).specialTestBit);
        assert.notEqual(cachedCredential, result3);
-     });
+     })
+      .timeout(5000);
```

Comment:
Curious. Why is this needed?

Reply:
This test will randomly time out from time to time.  I am dropping it from this PR and doing another.